src/commands/stats.ts

```diff
@@ -1,5 +1,9 @@
 import { getUpTime } from '../utilities/get-up-time';
 import { CommandDef } from './command-def';
+import getRepoInfo from 'git-repo-info';
+const info = getRepoInfo();
+import dotenv from 'dotenv';
+dotenv.config();
```

Comment:
`dotenv` should be ran earlier in the `index.ts` or `get-config` file. I think during the refactor, or earlier this wasn't being ran as early as possible. Instead of "hiding" this call away in this command, we should move it somewhere in the `index.ts` file (earlier in the imports, the "earlier" it runs)

Reply:
Actually, we can just run the `process.env.SOURCE_VERSION` inside the `get-config` file. Nice, I totally forgot about this

Reply:
Actually, it will work fine without importing it again. So deleting these 2 lines will be fine since it was called before.